blocks/sub-nav/sub-nav.css

```diff
@@ -27,12 +31,12 @@
   align-items: center;
   height: var(--sub-nav-height);
   width: 100%;
-  max-width: var(--nav-width);
   margin-inline: auto;
-  padding-inline-start: 3rem;
-  padding-inline-end: 5.5rem;
   font: 14px/40px var(--ff-subheadings-medium);
   position: relative;
+  max-width: var(--wrapper-width);
+  padding: 0;
+  border-top: solid 1px var(--c-secondary-silver);
```

Comment:
I added styles for stuck in-page nav. The problem is detecting when the in-page nav sticks the header. I used the scroll event that is already in the in-page nav but because of that, there is a delay with the styling. If you want I can create a new scroll event listener.